ha/setup/create_pacemaker_resources.py

```diff
@@ -333,3 +332,62 @@ def create_all_resources(cib_xml=const.CIB_FILE, push=True, **kwargs):
         Log.info("HA Rules: Successfully configured all HA resources and its configuration.")
     except Exception:
         raise CreateResourceError("Failed to populate cluster with resources")
+
+class ConfigAlertResource:
```

Comment:
can we create a separate file for alert configuration since its not actually a resource ?

Reply:
Updated